wp-content/themes/humanity-theme/theme.json

```diff
@@ -20,127 +20,82 @@
           "name": "White"
         },
         {
-          "slug": "amnesty-primary",
-          "color": "#fdff28",
+          "slug": "primary",
+          "color": "#fff",
           "name": "Primary"
         },
         {
-          "slug": "amnesty-primary-state",
-          "color": "#f8d72b",
+          "slug": "primary-state",
+          "color": "#000",
           "name": "Primary State"
         },
         {
-          "slug": "amnesty-secondary",
+          "slug": "secondary",
           "color": "#2196f3",
           "name": "Secondary"
         },
         {
-          "slug": "amnesty-accent",
+          "slug": "accent",
           "color": "#2196f3",
           "name": "Accent"
         },
         {
-          "slug": "amnesty-grey-brand",
-          "color": "#b3b3b3",
-          "name": "Grey Brand"
-        },
-        {
-          "slug": "amnesty-grey-xx-light",
+          "slug": "grey-lightest",
           "color": "#f7f7f7",
-          "name": "Grey Extra Extra Light"
+          "name": "Lightest Grey"
         },
         {
-          "slug": "amnesty-grey-x-light",
-          "color": "#eee",
-          "name": "Grey Extra Light"
-        },
-        {
-          "slug": "amnesty-grey-sm-light",
-          "color": "#e7e7e7",
-          "name": "Grey Small Light"
-        },
-        {
-          "slug": "amnesty-grey-light",
+          "slug": "grey-lighter",
           "color": "#e4e4e4",
-          "name": "Grey Light"
+          "name": "Lighter Grey"
         },
         {
-          "slug": "amnesty-grey-md-light",
-          "color": "#b5b5b5",
-          "name": "Grey Medium Light"
-        },
-        {
-          "slug": "amnesty-grey-mid-light",
-          "color": "#717272",
-          "name": "Grey Mid Light"
+          "slug": "grey-light",
+          "color": "#b3b3b3",
+          "name": "light Grey"
         },
         {
-          "slug": "amnesty-grey-base",
+          "slug": "grey-base",
           "color": "#767676",
-          "name": "Grey Base"
+          "name": "Default Grey"
         },
         {
-          "slug": "amnesty-grey-mid-dark",
+          "slug": "grey-dark",
           "color": "#525252",
-          "name": "Grey Mid Dark"
+          "name": "Dark Grey"
```

Comment:
Dark Grey and Dark are the same colour - Could this be reduced to one colour value?

Reply:
Good spot! I will take a further look at the dark colours.

Reply:
I actually couldn't see any of the "dark" colours in use anywhere, so I've removed all three in 7859106